docs/operator-manual/rbac.md

```diff
@@ -30,7 +30,28 @@ Breaking down the permissions definition differs slightly between applications a
 
 Resources: `clusters`, `projects`, `applications`, `repositories`, `certificates`, `accounts`, `gpgkeys`
 
-Actions: `get`, `create`, `update`, `delete`, `sync`, `override`, `action`
+Actions: `get`, `create`, `update`, `delete`, `sync`, `override`,
+`action/<path/to/action>`
+
+#### Application resources
+
+The resource path for application objects is of the form
+`<project-name>/<application-name>`.
```

Comment:
Can objects other than `application-name` be used? How would one reference a pod belonging to an application, for example?

Reply:
Based on [the policy validation regex](https://github.com/argoproj/argo-cd/blob/9bdf0bae2db6884f23b8ddb4b6dfc473b22ad6f3/pkg/apis/application/v1alpha1/types.go#L1600), I agree that only application names are allowed. No sub-resources.